drivers/timer/cavs_timer.c

```diff
@@ -195,7 +184,7 @@ uint32_t sys_clock_cycle_get_32(void)
 
 uint64_t sys_clock_cycle_get_64(void)
 {
-	return count64();
+	return count();
```

Comment:
nice. wondering if all other 64-bit timer functions in https://github.com/zephyrproject-rtos/zephyr/commit/918a574c88c3b06cfbb2c6e6382b15482834b647 are correct?